### PR TITLE
Add role to determine of root volume

### PR DESCRIPTION
**Pull request by sittisak**
Profab doesn't  determine root volume.
solution
- add file volume for create root volume
- use a AMI  with disk = 'ebs'
- add role on server file

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -152,6 +152,14 @@ Sets the instance size to be launched to the requested size. Current valid sizes
 
 Unlike the normal EC2 default, profab has a default size of `t1.micro`.
 
+## volume ##
```
Comment (KayEss) [inline README.markdown]: This seems like a bad name for something that appears to change the root volume size. Something longer and more descriptive is probably needed: `root_volume_size` maybe

File: profab/role/ami/ubuntu.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ class Configure(Role):
     def ami(self, region, bits, size):
         """Return the AMI that was passed in to the role.
         """
-        disk = 'ebs' if size == 't1.micro' else 'instance'
```
Comment (KayEss) [inline profab/role/ami/ubuntu.py:19]: This old code is wrong, but I'm not sure the new code is better. Is always defaulting to EBS the right thing?

**Review by KayEss — CHANGES_REQUESTED** [review]

File: profab/role/ami/ubuntu.py (hunk 2 above)
Comment (sittisak) [reply profab/role/ami/ubuntu.py:19]: Infra guy suggest me to do that, infra guy said that EBS can resize root volume but instance can't resize root volume.

Comment (KayEss) [reply profab/role/ami/ubuntu.py:19]: That's not really an argument for why EBS being the default is the right thing to do. If the user wants an instance disk how do they do that?

Comment (sittisak) [reply profab/role/ami/ubuntu.py:19]: "Is always defaulting to EBS the right thing?"
In that time, they hard coded to make it work with our UAT environment.
UAT instance is R4.xlarge

**Pull request merged**